src/reference/modules/installation/pages/versions.adoc

```diff
@@ -2,19 +2,18 @@
-* *`Y` or `X` version increments* are *major* releases. These versions include new features, and the amount of changed code can be considerable. An upgrade path from one major version to next will always be provided, but may not be between two non-consecutive versions (ie, upgrading from 4.1 to 4.2 is supported, and from 4.2 to 4.3 is too, but upgrading directly from 4.1 to 4.3 may not be – check the xref:reference:installation:upgrade.adoc[upgrade section] of the user manual for specifics). They are released every few months.
+* *`X` version increments* are *major* releases. These versions include new features and architecture changes, and the amount of modifications can be considerable. An upgrade path from one major version to the next will always be provided (i.e., upgrading from 5 to 6 is supported, and from 6 to 7 is too, but upgrading directly from 5 to 7 will generally not be – check the xref:reference:installation:upgrade.adoc[upgrade section] of the user manual for specifics). They are released approximately every 9 to 12 months.
+* *`Y` version increments* are *minor* releases. These versions include small new features and various improvements, but no major change in the architecture of Rudder. An upgrade path from one minor version to the next ones (for a given major version) will be provided – check the xref:reference:installation:upgrade.adoc[upgrade section] of the user manual for specifics). They are released every few months.
+* *`Z` version increments* are *patch* releases. They contains bug or security fixes, and we make a lot of effort to reduce the amount of changes that go into these version (a full change log is always available). Upgrading your Rudder version to the latest patch release is always recommended, and is designed to be easy. They are released every few weeks.
 
 == Rudder life cycle
 
 A maintained version is a major version for which we release minor bug fix releases and communicate about security vulnerabilities.
 
-* Our general policy is to *maintain major releases until 3 months after the next major version* is released to give some time for upgrading.
-* For a slower major upgrade pace, Rudder subscription provides (among other benefits) *long-term maintenance of 18 to 24 months after release* (depending on the subscription level) for Rudder versions, with a minimal upgrade window of 6 to 9 months. You can read more about it on the https://www.rudder.io/en/pricing/subscription/[subscription page].
-
-Note: The ESR tag was previously given to major releases that were maintained 6 months after the next ESR version was announced. Rudder 4.1 is the last ESR release, extended maintenance is now provided as part of https://www.rudder.io/en/pricing/subscription/[Rudder subscription].
+* Our general policy is to *provide patch releases for an X.Y version during 3 months after the next minor or major version* is released to give some time for upgrading.
```

Comment:
```suggestion
* Our general policy is to provide *patch releases* for a minor version during *3 months* following next *minor* version release, giving some time to upgrade. That period of time is extended to *9 months* when we release a new major version, because that upgrade can be more complex and may need some preparation. It will also give more time for the new major version to be more mature.
```

Reply:
I propose to just change to "at least 3 months". It gives a lower bar and fixes expectation, but let us adapt in case a major release is way to unstable during 3 firsts months (but the idea is clearly to make community switches early and help us test the new version)

Reply:
(and we don't force anyone to update during that period, we just stop publishing patch release)